mv3/python/export.py

```diff
@@ -8,39 +8,54 @@
 import torch
 import torchvision.models as models
 from executorch.backends.apple.coreml.partition import CoreMLPartitioner
-from executorch.backends.apple.mps.partition import MPSPartitioner
-from executorch.exir.backend.backend_details import CompileSpec 
+from executorch.exir import to_edge 
 from executorch.backends.xnnpack.partition.xnnpack_partitioner import XnnpackPartitioner
 from executorch.exir import to_edge_transform_and_lower
-import ssl
-import certifi
 
 
 def main() -> None:
     model = models.mobilenet_v3_small(weights="DEFAULT").eval()
     sample_inputs = (torch.randn(1, 3, 224, 224),)
 
+    et_program_portable = to_edge(
+        torch.export.export(model, sample_inputs),
+    ).to_executorch() 
+
     et_program_coreml = to_edge_transform_and_lower(
         torch.export.export(model, sample_inputs),
         partitioner=[CoreMLPartitioner()],
     ).to_executorch()
 
-    et_program_mps = to_edge_transform_and_lower(
+    
+    # MPS backend doesn't work yet with pip install today.
+    # Currently, it is just falling back to portable ops instead.
+    # 
+    # Please install ExecuTorch from source if you want to run to MPS backend:
+    # 
+    # https://pytorch.org/executorch/main/using-executorch-building-from-source.html#environment-setup
+    # ./install_executorch.sh --pybind mps coreml xnnpack
+    # ./backends/apple/mps/install_requirements.sh
+    # 
+    # et_program_mps = to_edge_transform_and_lower(
+    #   torch.export.export(model, sample_inputs),
+    #   partitioner=[MPSPartitioner([CompileSpec("use_fp16", bytes([True]))])], 
+    # ).to_executorch()
+    et_program_mps = to_edge(
         torch.export.export(model, sample_inputs),
-        partitioner=[MPSPartitioner([CompileSpec("use_fp16", bytes([True]))])], 
     ).to_executorch()
 
     et_program_xnnpack = to_edge_transform_and_lower(
         torch.export.export(model, sample_inputs),
         partitioner=[XnnpackPartitioner()],
     ).to_executorch()
 
-    save_path = "executorch-examples/mv3/apple/ExecuTorchDemo/ExecuTorchDemo/Resources/Models/MobileNet/"
-    with open(save_path+"mv3_coreml_all.pte", "wb") as file:
+    with open("mv3.pte", "wb") as file:
+        et_program_portable.write_to_file(file)
```

Comment:
Does this echo that the file was written? If not, it might improve DevEx if we can echo to stderr that `Model written: x`